pages/documentation/developer/tutorials/sensitivity-analysis.md

```diff
@@ -332,18 +332,17 @@ You can check the content of the file `src/main/resources/factors.json` to verif
 It is possible to overwrite the Hades2 sensitivity parameters with a JSON file, which we'll do now:
 ```java
 Path parametersFile = Paths.get(SensitivityTutorialComplete.class.getResource("/sensi_parameters.json").getPath());
-SensitivityComputationParameters params = SensitivityComputationParameters.load();
-JsonSensitivityComputationParameters.update(params, parametersFile);
+SensitivityAnalysisParameters params = SensitivityAnalysisParameters.load();
+JsonSensitivityAnalysisParameters.update(params, parametersFile);
 ```
 We first create sensitivity analysis parameters by loading the YML configuration file, and then update them based on the provided JSON file for sensitivity parameters.
 
 ## Run a set of sensitivity analyses on all network states
 
 We can now run all the sensitivity analyses at once, through:
 ```java
-SensitivityComputationResults systematicSensiResults = sensitivityComputation.run(
-    jsonFactorsProvider, contingenciesProvider, VariantManagerConstants.INITIAL_VARIANT_ID,
-    params).join();
+SensitivityAnalysisResults systematicSensiResults = SensitivityAnalysis.find().run(network,
```

Comment:
Remove the extra call to the `find()` method, and maybe put all the parameters on a single line if the rendering is OK